src/lib/widgets/now-playing/ui/now-playing.svelte

```diff
@@ -30,51 +27,73 @@
     const i = speedValues.indexOf($speed);
     $speed = speedValues[(i + 1) % speedValues.length];
   }
+
+  onMount(lockScrollWhileMounted);
 </script>
 
-<div
-  class="bg-black bg-opacity-20 pt-4 flex flex-col justify-end absolute bottom-0"
-  transition:fly={{ y: 100 }}
->
-  <div class="bg-slate-800 flex flex-col rounded-t-2xl py-10 px-8 items-center">
-    <div class="flex flex-col items-center relative">
-      {#if transcriptShown}
-        <Transcript {episode} class="absolute top-0 left-0 w-full h-full bg-slate-800 px-0.5" />
-      {/if}
-      <div class="flex flex-col items-center" class:invisible={transcriptShown}>
-        <EpisodeDisplay {episode} {podcast} />
+{#if $currentlyPlayingEpisode !== null}
+  <div
+    class="bg-black bg-opacity-20 flex flex-col justify-end fixed bottom-0 left-0 h-screen"
```

Comment:
Is there a way to center it in the screen instead of aligning it to `left-0`?

Reply:
What for?

Reply:
Because this looks ugly:
![image](https://user-images.githubusercontent.com/11016151/160236892-056e9ce5-829d-452b-b2e2-24af026aeeff.png)